packages/experimental/Shimmer/src/Shimmer.types.win32.ts

```diff
@@ -26,7 +26,14 @@ export interface ShimmerWaveProps extends ShimmerTokens {
   viewBoxWidth: number;
 }
 
-export interface ShimmerSlotProps extends ShimmerProps {
+export interface ShimmerProps extends ViewProps, ShimmerTokens {
```

Comment:
ShimmerProps and ShimmerType are same definition as .ts file, can we use the same and remove from .win32 file.

Reply:
Was initially not sure if I should make elements a required prop for win32 similar to what I did for other plats. Have made the change now.